CHANGELOG.md

```diff
@@ -16,6 +16,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 ### Security
 
+## [2.1.1-DEV1] - 2024-05-12
```

Comment:
The changes have to be part of the section above.
The `Unreleased` and `timespamp` will be replaced in the build process with the actual data. See [here](https://betonquest.org/DEV/Documentation/CHANGELOG/#212-dev-2-2024-05-09).
The rest looks great :)